piano-hero/mobile/src/main/java/com/novoda/pianohero/GameMvp.java

```diff
@@ -0,0 +1,38 @@
+package com.novoda.pianohero;
+
+interface GameMvp {
+
+    interface Model {
+
+        RoundViewModel startGame();
+
+        RoundViewModel onNotesPlayed(Callback callback, Note... note);
+
+        interface Callback {
+            void onSequenceComplete();
+        }
+    }
+
+    interface View {
+
+        void showRound(RoundViewModel viewModel);
+
+        void showGameComplete();
+
+        void showError();
+
+        void showKeyboard();
```

Comment:
heh but now Ataul has spoken to me, he's convinced me of your original comment because this is all view layer without any logic in the model, so I've inlined this back into the activity